integration-tests/run_tests

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Anything bash specific in here?

Reply:
Well, there's `[[` and `pipefail`. Bash is widespread and less painful than POSIX shell.

Reply:
FYI this change is just to make it straightforward for devs to run integration tests, not to be too much universal. It used to depend on Bash previously too.

Reply:
> Bash is widespread and less painful than POSIX shell.
... and slow. But, anyway, you're right - ifs are non-POSIXy etc. Got ya, approved :-)